pinot-tools/src/main/resources/examples/batch/starbucksStores/starbucksStores_offline_table_config.json

```diff
@@ -15,8 +15,11 @@
     "loadMode": "MMAP",
     "invertedIndexColumns": [
     ],
-    "h3IndexColumns": [
-      "location_st_point"
+    "h3IndexConfigs": [
```

Comment:
I think we added a new way of adding indexing configs for each field... can you follow the pattern used by text index? can be done in another PR